tests/e2e-instrumentation/instrumentation-select/01-install-app-select.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
```

Comment:
Can you also add a second Pod here without the label, and assert that it doesn't get instrumentation injected? You can use chainsaw to simply check that the container count is 1 without checking the full manifest.

Reply:
Done.